Library/Formula/syncthing-inotify.rb

```diff
@@ -0,0 +1,63 @@
+class SyncthingInotify < Formula
+  desc "File watcher intended for use with Syncthing"
+  homepage "https://github.com/syncthing/syncthing-inotify"
+  url "https://github.com/syncthing/syncthing-inotify.git",
+    :tag => "v0.6.5", :revision => "2b290b929e18e7cd1cfea11240e4fa870d6638d5"
+
+  head "https://github.com/syncthing/syncthing-inotify.git"
+
+  bottle do
+    cellar :any
+    sha256 "392d1b861c5eae39e44148842947dfbeb3640cb0d031e0a42fef3ec49d31252b" => :yosemite
+    sha256 "28b78c3826f3a9a6c1b521ad91b2e7c91b3bc6417f7e175a2e83960ad2f78e1d" => :mavericks
+    sha256 "a1c9ae1947ae60898b14820577ffafc4bbb5e92039d02ebf9e8d2aa9e5fffaea" => :mountain_lion
+  end
```

Comment:
Remove this; not sure where you got these values for but they will need to be built by us and added after this formula is included.